openadapt/models.py

```diff
@@ -526,7 +545,16 @@ def get_active_window_event(
         Returns:
             (WindowEvent) the active window event.
         """
-        return WindowEvent(**window.get_active_window_data(include_window_data))
+        window_event_data = window.get_active_window_data(include_window_data)
+
+        a11y_event_data = window_event_data.get("state", {}).copy()
+        window_event_data.pop("state", None)
```

Comment:
Is this still relevant?

Reply:
I tried without this logic and while replaying I was getting `TypeError: 'state' is an invalid keyword argument for WindowEvent`

Reply:
But you removed WindowEvent.state here: https://github.com/OpenAdaptAI/OpenAdapt/pull/842/files#diff-224d5ce89a18f796cae99bf3da5a9862def2127db2ed38e68a07a25a8624166fL503

Reply:
Let's please rename everywhere window "state" to a11y_data, and `config.RECORD_WINDOW_DATA` to `config.RECORD_A11Y_DATA`.